bot/utils/time.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+
+from dateutil.relativedelta import relativedelta
+
+
+# All these functions are from https://github.com/python-discord/bot/blob/master/bot/utils/time.py
```

Comment:
I don't think this comment is needed.

Reply:
I don't see the harm of having the original source linked, so I'm inclined to keep it.